numpy/ma/core.py

```diff
@@ -4240,22 +4240,33 @@ def get_real(self):
         return result
     real = property(fget=get_real, doc="Real part")
 
-    def count(self, axis=None):
+    def count(self, axis=None, keepdims=np._NoValue):
         """
         Count the non-masked elements of the array along the given axis.
 
         Parameters
         ----------
-        axis : int, optional
-            Axis along which to count the non-masked elements. If `axis` is
-            `None`, all non-masked elements are counted.
+        axis : None or int or tuple of ints, optional
+            Axis or axes along which the count is performed.
+            The default (`axis` = `None`) is perform the count sum over all
```

Comment:
According to [the docs](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard) as I read them, single-backticks go around variable, module, function, and class names, while double-backticks go around inline code. So I think it's right here.

I'll fix the grammar though.